metals/src/main/scala/scala/meta/internal/metals/MetalsLspService.scala

```diff
@@ -382,15 +382,17 @@ class MetalsLspService(
     )
   )
 
+  private val connectionBspStatus =
+    new ConnectionBspStatus(bspStatus, folder, clientConfig.icons())
+
   private val bspErrorHandler: BspErrorHandler =
     new BspErrorHandler(
-      languageClient,
-      folder,
       () => bspSession,
       tables,
+      connectionBspStatus,
     )
 
-  private val buildClient: ForwardingMetalsBuildClient =
+  val buildClient: ForwardingMetalsBuildClient =
```

Comment:
```suggestion
  private val buildClient: ForwardingMetalsBuildClient =
```

there doesn't seem to be any need for the change anymore.

Reply:
Used in test.